articles/quickstarts/install-binder.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Develop with Q# and Binder
+description: Learn how to create a Q# application using Binder.
+author: bradben
+ms.author: v-benbra
+ms.date: 9/03/2020
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.install.binder
+no-loc: ['Q#', '$$v']
+---
+# Develop with Q# and Binder
+
+You can use Binder to run your Jupyter Notebooks online.
+
+To configure Binder automatically and experiment with the QDK samples:
+
+- Open a browser and run https://aka.ms/try-qsharp.
+
+To configure Binder manually:
+
+1. Go to [mybinder.org](https://mybinder.org).
+1. Fill out the fields, using *microsoft/quantum* as the GitHub URL.
+
+![MyBinder form](~/media/mybinder.png)
+    
+## Next steps
+
+Now that you have set up your Binder environment, you can write and run [your first quantum program](xref:microsoft.quantum.quickstarts.qrng).
```

Comment:
The linked page here doesn't really have any information about the Jupyter Notebook environment that would be relevant to a first-time user.

Since they have just launched the samples repo in Binder, would it make sense instead to put a screenshot of the Binder landing page and suggest that they click on "Q# notebook" under "Getting started"?

![image](https://user-images.githubusercontent.com/3620100/92271664-105c9300-ee9d-11ea-9e34-d651f471766c.png)

Reply:
Updated Binder topic per Russ and Chris feedback: Option for automatic install using QDK samples, and option for user to config Binder for their own repo.